next-docs/pages/templates/typescript/contracts.mdx

```diff
@@ -1 +1,453 @@
-# Contracts
+# Contracts (Vanilla and React)
+
+This section explains the contracts for the vanilla and react templates.
+The Three.JS template uses diffeerent contracts.
+
+The onchain components can be divided into two types of functionality:
+
+- **Data**, the part of the system that stores the information.
+- **Logic**, the part of the system that can be called to read or modify the data.
+
+## Data
+
+### The data schema
+
+#### `mud.config.ts`
+
+The data schema is declared in `packages/contracts/mud.config.ts`.
+[Read more details about the schema definition here](/world/config#global-configuration-keys).
```

Comment:
we'll have to update this link to the matching place in the new docs once it's ready

Reply:
actually, let's just point it to the new place already
```suggestion
[Read more details about the schema definition here](/cli/config).
```